imap/cyr_dbtool.c

```diff
@@ -305,6 +305,11 @@ int main(int argc, char *argv[])
 
         fprintf(stderr, "\n");
         fprintf(stderr, "\n");
+        fprintf(stderr, "Options:\n");
+        fprintf(stderr, "  -c     convert database to named backend if not already\n");
```

Comment:
The man page for this tool doesn't bother to mention this option either.  There's a separate "cvt_cyrusdb" tool for this specific task anyway, so I'm kind of inclined to leave this option out, cause I agree converting in place looks dangerous.

Reply:
Yeah, sounds right to me.